grq2/es_connection.py

```diff
@@ -44,5 +51,9 @@ def get_grq_es(logger=None):
                 retry_on_timeout=True,
             )
         else:
-            GRQ_ES = ElasticsearchUtility(es_url, logger)
+            if es_cluster_mode:
+                hosts = [app.conf.JOBS_ES_URL, app.conf.GRQ_ES_URL, app.conf.METRICS_ES_URL]
+            else:
+                hosts = [app.conf.JOBS_ES_URL]
```

Comment:
Should this fall back to what it was before?

```suggestion
                hosts = [es_url]
```